.samples/hello-world-dotnet-webapp.yaml

```diff
@@ -0,0 +1,9 @@
+displayName: Hello World Web App in .NET
+description: Print Hello World message
```

Comment:
Let's make the description more meaningful.

Reply:
Fixed with https://github.com/wso2/choreo-samples/pull/101/commits/38110a68153068ccabca2a2bdf30142179c2215f